environments.json

```diff
@@ -253,6 +253,35 @@
     "status": "Stable",
     "version": "1.31.1"
   }
+],[
+  {
```

Comment:
We should only keep dotnet8 entry here and remove rest of entries probably.

Reply:
remove dotnet20 and dotnet from this file and others? or just this file?